### PR TITLE
Add Extentions and Search engines catagory

### PR DESCRIPTION
**Pull request by nonetrix**
Many ports are under browser in my option this should only be used for web browsers

### CODE REVIEW

**Comment by nekowinston** [comment]
I absolutely agree with that. I just didn't want to do it on my own without checking with the rest of @catppuccin/core first.

File: README.md [hunk 1]
```diff
@@ -917,7 +917,7 @@ Catppuccin is available for various apps and in different formats. Here is a lis
 -   [Duckduckgo](https://github.com/catppuccin/duckduckgo)
 -   [Whoogle](https://github.com/catppuccin/whoogle)
 
-### Extentions
+#### Extentions
```
Comment (nekowinston) [inline README.md:920]: Still spelling "Extensions" wrong.

**Comment by nonetrix** [comment]
🥲

**Comment by nekowinston** [comment]
Apologies, I didn't know that I have write permissions on the PR branch.

LGTM.
cc: @Pocco81

**Comment by Pocco81** [comment]
Seems appropriate 👍

**Pull request merged**